GooglePlayInstant/InstallLauncher.cs

```diff
@@ -15,8 +15,7 @@
 using System;
 using UnityEngine;
 
-namespace GooglePlayInstant
-{
+namespace GooglePlayInstant {
```

Comment:
All of these style changes (braces, spaces) are inconsistent with the existing code base.

Is it possible for you to use Rider and it's "Code -> Reformat Code" option for style modification?